src/main/java/org/jabref/logic/util/BuildInfo.java

```diff
@@ -49,10 +51,22 @@ public BuildInfo(String path) {
         azureInstrumentationKey = properties.getProperty("azureInstrumentationKey", "");
         minRequiredJavaVersion = properties.getProperty("minRequiredJavaVersion", "1.8");
         allowJava9 = "true".equals(properties.getProperty("allowJava9", ""));
+        try {
+            releaseDate = Version.getAllAvailableVersionsDate().get(0);
+        } catch (IOException e) {
+            releaseDate = "";
+        }
     }
 
     public Version getVersion() {
-        return version;
+        List<Version> list = null;
+        try {
+            list = Version.getAllAvailableVersions();
```

Comment:
Better check if the list is empty isEmpty(), using Exceptions for control flow handling is not a good idea.